File: ios/Classes/TorPlugin.swift
//
//  SystemProxyPlugin.swift
//  tor
//
//  Created by w on 2025/10/28.
//

import Flutter
import UIKit

public class TorPlugin: NSObject, FlutterPlugin {
    public static func register(with registrar: FlutterPluginRegistrar) {
        SystemProxyPlugin.register(with: registrar)
    }
}

File: android/src/main/kotlin/com/oxchat/tor/TorPlugin.kt
package com.oxchat.tor

import androidx.annotation.NonNull
import io.flutter.embedding.engine.plugins.FlutterPlugin

class TorPlugin : FlutterPlugin {
    override fun onAttachedToEngine(@NonNull flutterPluginBinding: FlutterPlugin.FlutterPluginBinding) {
        SystemProxyPlugin().onAttachedToEngine(flutterPluginBinding)
    }

    override fun onDetachedFromEngine(@NonNull binding: FlutterPlugin.FlutterPluginBinding) {}
}